OpenEdXMobile/res/layout/fragment_course_unit_grade.xml

```diff
@@ -1,55 +1,60 @@
-        tools:text="@string/assessment_not_available"
-        android:textAlignment="center"
-        android:textColor="@color/edx_brand_gray_base"
-        android:textSize="@dimen/edx_base"
-        tools:targetApi="17" />
+        android:orientation="vertical">
 
-    <TextView
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:layout_marginBottom="20dp"
-        android:layout_marginLeft="60dp"
-        android:layout_marginRight="60dp"
-        android:gravity="center"
-        android:singleLine="false"
-        android:text="@string/assessment_explore_on_web"
-        android:textAlignment="center"
-        android:textColor="@color/edx_brand_gray_base"
-        android:textSize="@dimen/edx_base"
-        tools:targetApi="17" />
+        <org.edx.mobile.view.custom.IconImageViewXml
+            android:id="@+id/content_error_icon"
+            android:layout_width="@dimen/unit_unavailable_error_icon_size"
+            android:layout_height="@dimen/unit_unavailable_error_icon_size"
+            android:layout_margin="@dimen/widget_margin_double"
+            app:iconColor="@color/edx_brand_gray_back"
+            app:iconName="fa-laptop" />
+
+        <TextView
+            android:id="@+id/not_available_message"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginLeft="@dimen/edx_quadruple_margin"
+            android:layout_marginRight="@dimen/edx_quadruple_margin"
+            android:layout_marginBottom="@dimen/widget_margin_double"
+            android:gravity="center"
+            android:singleLine="false"
+            android:textAlignment="center"
+            android:textColor="@color/edx_brand_gray_base"
+            android:textSize="@dimen/edx_base"
+            tools:targetApi="17"
+            tools:text="@string/assessment_not_available" />
+
+        <TextView
+            android:id="@+id/not_available_message_2"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginLeft="@dimen/edx_quadruple_margin"
+            android:layout_marginRight="@dimen/edx_quadruple_margin"
+            android:layout_marginBottom="@dimen/widget_margin_double"
+            android:gravity="center"
+            android:singleLine="false"
+            android:text="@string/assessment_explore_on_web"
+            android:textAlignment="center"
+            android:textColor="@color/edx_brand_gray_base"
+            android:textSize="@dimen/edx_base"
+            tools:targetApi="17" />
 
-    <Button
-        android:id="@+id/view_on_web_button"
-        style="@style/edX.Widget.RegularButton"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_marginBottom="20dp"
-        android:text="@string/assessment_view_on_web" />
+        <Button
```

Comment:
Are we need to show the `View on Web` button? because the use can't access the course section on web as well  that have `authorization_denial_reason` key.

Reply:
Yes, we need it, cuz on the web the user can then pay for the verified track and access the specific component.